test/services/bill-runs/determine-blocking-bill-run.service.test.js

```diff
@@ -22,15 +22,18 @@ describe('Determine Blocking Bill Run service', () => {
   let regionId
   let season
   let toFinancialYearEnding
+  let billRunId
+  let billRunIdTwo
 
   beforeEach(async () => {
-    await DatabaseSupport.clean()
-
     const { endDate } = determineCurrentFinancialYear()
     toFinancialYearEnding = endDate.getFullYear()
 
     const region = await RegionHelper.add()
     regionId = region.id
+
+    billRunId = generateUUID()
```

Comment:
I think this is a good example / case for generating the  uuid in the code where they were before. A minimal touch has fixed this test.